WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -138,32 +126,42 @@ private extension TotalsView {
                 PointOfSaleCardPresentPaymentInLineMessage(messageType: inlinePaymentMessage)
             } else {
                 Text("Reader connected")
+                Button(action: viewModel.cardPaymentTapped) {
+                    Text("Collect Payment")
+                }
             }
         case .disconnected:
+            Text("Reader disconnected")
             Button(action: viewModel.cardPaymentTapped) {
                 Text("Collect Payment")
             }
         }
     }
 
-    @ViewBuilder func priceFieldView(title: String, formattedPrice: String) -> some View {
-        VStack(alignment: .leading, spacing: .zero) {
+    @ViewBuilder func priceFieldView(title: String, formattedPrice: String?, shimmeringActive: Bool) -> some View {
+        HStack(alignment: .center, spacing: .zero) {
             Text(title)
-            Text(formattedPrice)
-                .font(.title2)
-                .fontWeight(.medium)
+                .font(Font.system(size: 20))
+            Spacer()
+            Text(formattedPrice ?? "-----")
+                .font(Font.system(size: 20))
+                .redacted(reason: formattedPrice == nil ? [.placeholder] : [])
+                .shimmering(active: shimmeringActive)
```

Comment:
I noticed that after the order is created, the taxes and total stay shimmering even after they show the unredacted detail.

I think two things are happening here:
1. We're not resetting `viewModel.isSyncingOrder` until the payment completes, which is unexpected, but it's because we `await` the `totalsViewWillAppear()` call within syncOrder. We can reset the flag outside the defer to resolve this.
2. However... we probably don't want different conditions for the value to be shimmering vs redacted in this case. Can we just use `formattedPrice == nil` for both?